source/exec.tex

```diff
@@ -0,0 +1,4872 @@
+transform_sender(
+  @\exposid{query-or-default}@(get_domain, sch, default_domain()),
+  @\exposid{make-sender}@(starts_on, sch, sndr))
+\end{codeblock}
+except that \tcode{sch} is evaluated only once.
+
+\pnum
+Let \tcode{out_sndr} and \tcode{env} be subexpressions
+such that \tcode{OutSndr} is \tcode{decltype((out_sndr))}.
+If \tcode{\exposconcept{sender-for}<Out\-Sndr, starts_on_t>} is \tcode{false},
+then the expressions \tcode{starts_on.transform_env(out_sndr, env)} and\linebreak
+\tcode{starts_on.transform_sender(out_sndr, env)} are ill-formed; otherwise
+\begin{itemize}
+\item
+\tcode{starts_on.transform_env(out_sndr, env)} is equivalent to:
+\begin{codeblock}
+auto&& [_, sch, _] = out_sndr;
+return @\exposid{JOIN-ENV}@(@\exposid{SCHED-ENV}@(sch), @\exposid{FWD-ENV}@(env));
+\end{codeblock}
+\item
+\tcode{starts_on.transform_sender(out_sndr, env)} is equivalent to:
+\begin{codeblock}
+auto&& [_, sch, sndr] = out_sndr;
+return let_value(
+  schedule(sch),
+  [sndr = std::forward_like<OutSndr>(sndr)]() mutable
+    noexcept(is_nothrow_move_constructible_v) {
+    return std::move(sndr);
+  });
+\end{codeblock}
+\end{itemize}
+
+\pnum
+Let \tcode{out_sndr} be a subexpression denoting
+a sender returned from \tcode{starts_on(sch, sndr)} or one equal to such, and
+let \tcode{OutSndr} be the type \tcode{decltype((out_sndr))}.
+Let \tcode{out_rcvr} be a subexpression denoting a receiver
+that has an environment of type \tcode{Env}
+such that \tcode{\libconcept{sender_in}<OutSndr, Env>} is \tcode{true}.
+Let \tcode{op} be an lvalue referring to the operation state
+that results from connecting \tcode{out_sndr} with \tcode{out_rcvr}.
+Calling \tcode{start(op)} shall start \tcode{sndr}
+on an execution agent of the associated execution resource of \tcode{sch}.
+If scheduling onto \tcode{sch} fails,
+an error completion on \tcode{out_rcvr} shall be executed
+on an unspecified execution agent.
+
+\rSec3[exec.continues.on]{\tcode{execution::continues_on}}
+
+\pnum
+\tcode{continues_on} adapts a sender into one
+that completes on the specified scheduler.
+
+\pnum
+The name \tcode{continues_on} denotes a pipeable sender adaptor object.
+For subexpressions \tcode{sch} and \tcode{sndr},
+if \tcode{decltype((sch))} does not satisfy \libconcept{scheduler}, or
+\tcode{decltype((sndr))} does not satisfy \libconcept{sender},
+\tcode{continues_on(sndr, sch)} is ill-formed.
+
+\pnum
+Otherwise,
+the expression \tcode{continues_on(sndr, sch)} is expression-equivalent to:
+\begin{codeblock}
+transform_sender(@\exposid{get-domain-early}@(sndr), @\exposid{make-sender}@(continues_on, sch, sndr))
+\end{codeblock}
+except that \tcode{sndr} is evaluated only once.
+
+\pnum
+The exposition-only class template \exposid{impls-for}
+is specialized for \tcode{continues_on_t} as follows:
+\begin{codeblock}
+namespace std::execution {
+  template<>
+  struct @\exposid{impls-for}@<continues_on_t> : @\exposid{default-impls}@ {
+    static constexpr auto get_attrs =
```

Comment:
```suggestion
    static constexpr auto @\exposid{get-attrs}@ =
```

Reply:
Same problem for [exec.run.loop.general] push-back and pop-front 
Both issues are present in the incoming paper.